crates/cli/src/bin/wasm-bindgen.rs

```diff
@@ -1,34 +1,52 @@
-use anyhow::{bail, Error};
-use clap::Parser;
+use anyhow::Error;
+use clap::{Parser, ValueEnum};
 use std::path::PathBuf;
 use std::process;
 use wasm_bindgen_cli_support::{Bindgen, EncodeInto};
 
+#[derive(Debug, Clone, ValueEnum)]
+#[clap(rename_all = "kebab-case")]
+enum Target {
+    Bundler,
+    Web,
+    Nodejs,
+    NoModules,
+    Deno,
+    ExperimentalNodejsModule,
+    Module,
+}
+
 #[derive(Debug, Parser)]
 #[command(
     name = "wasm-bindgen",
     version,
     about,
-    long_about = None,
-    after_help = "Additional documentation: https://wasm-bindgen.github.io/wasm-bindgen/reference/cli.html",
+    after_help = "Additional documentation: https://wasm-bindgen.github.io/wasm-bindgen/reference/cli.html"
 )]
+#[group(id = "target-group", multiple = false)]
```

Comment:
I'm not to familiar with `clap`. Interesting requirements to disable multiple arguments.

Reply:
I see you already removed that.